pay-api/src/pay_api/models/invoice.py

```diff
@@ -173,3 +178,39 @@ def _clean_up(self, data, many):  # pylint: disable=unused-argument
             data['status_code'] = PaymentStatus.COMPLETED.value
 
         return data
+
+
+@define
+class InvoiceSearchModel:  # pylint: disable=too-few-public-methods, too-many-instance-attributes
+    """Main schema used to serialize invoice searches, plus csv / pdf export."""
+
+    id: int
+    corp_type_code: str  # TODO probably a good idea to move this.
```

Comment:
This will require some looking around to fix, as there are many different paths to this object.